src/main/groovy/com/athaydes/spockframework/report/vivid/VividAstInspector.groovy

```diff
@@ -158,6 +152,16 @@ class VividASTVisitor extends ClassCodeVisitorSupport {
         visitStatements = previousIsTestMethod
     }
 
+    @Override // This is overridden to avoid visiting annotations.
+    void visitAnnotations(AnnotatedNode node) {
+        for (AnnotationNode annotation : node.getAnnotations()) {
```

Comment:
In Groovy, please use `for (def annotation in node.annotations)` as thats' much nicer to look  at :)

Reply:
I agree! The code was copied from the overridden method, but it makes sense to use a more concise syntax here.